postinstall.one

```diff
@@ -32,7 +32,7 @@ then
 fi
 
 if [ -f /etc/init.d/one-context ]; then
-    if [ -d /etc/sysconfig/network-scripts ]; then
+    if [ -d /etc/sysconfig/network-scripts ] || [ -d /etc/sysconfig/network ]; then
```

Comment:
This is probably not relevant with the openSUSE, as it's using the systemd (and no SysV scripts should be deployed in /etc/init.d/).